### ISSUE TITLE
[Bug-Candidate]: Echidna reproducers can have reverting transactions

### ISSUE THREAD

**Issue by glarregay-tob** [crytic/fuzz-utils#33]
### Describe the issue:

It is possible for echidna reproducers or corpus entries to have transactions that revert. When those transactions are converted to a foundry test, the whole test fails.

### Code example to reproduce the issue:

It requires quite a bit of "luck" to generate one of those call sequences, as they are usually optimized during shrinking. But using a low value for the shrinking runs can help finding one, I guess.

### Version:

commit [4db6306](https://github.com/crytic/fuzz-utils/commit/4db63060b8613720a18d8a799a7ff6e8dce10229) + PR #32

**Comment by tuturu-tech** [comment]
Good point, although I'm unsure how to resolve this since Echidna/Medusa sequences don't contain any info on if the call reverted or not. We could ignore reverts by wrapping the call in a try/catch, albeit that would inflate the LOC a bit
